docs/guide.md

```diff
@@ -334,6 +334,42 @@ the context of a `Session` with the `DataClient`:
 >>> asyncio.run(main())
 ```
 
+#### Downloading an Asset
+
+Downloading an asset is a multi-step process involving: activating the asset,
+waiting for the asset to be active, downloading the asset, and, optionally,
+validating the downloaded file.
+
+
+With wait and download, it is often desired to track progress as these
+processes can take a long time. Therefore, in this example, we use a simple 
+print command to report wait status. `download_asset` has reporting built in.
+
+```python
+>>> async def download_and_validate():
```

Comment:
I am against having the added "`>>>`"s and "`...`"s in example code because it makes it difficult for users to copy and paste and immediately use. We're also inconsistent with its usage. We add it to many of our examples, but don't have it in the checksum example (line 262 in this MD).

Reply:
I'm against it too. It is included here for consistency and we need to do a sweep of the docs to remove these. I'm not sure why it isn't in the checksum example, it probably slipped through a PR. But that isn't a part of this PR.

Reply:
The tricky thing is that `>>>` is kinda normalized in Python and numpy. See https://docs.python.org/3/tutorial/introduction.html#numbers and https://numpydoc.readthedocs.io/en/latest/format.html#examples. So without some doc linting, this usage might continue to creep in. For what it's worth, the Python doc source includes the `>>>` prompt (see https://github.com/python/cpython/blob/3.10/Doc/tutorial/introduction.rst#numbers) and the website has a js widget to hide it if you want to copy it without.